releaseNotes/MicrosoftAzureAppConfigurationAspNetCore.md

```diff
@@ -1,6 +1,10 @@
 ## Microsoft.Azure.AppConfiguration.AspNetCore
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Azure.AppConfiguration.AspNetCore)
 
+### 4.0.0-preview - July 23, 2020
+* Added multi-targeting support for .NET Core 3.1. [#173](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/173)
```

Comment:
How about this?

Added multi-targeting support for .NET Core 3.1 *besides .NET Standard 2.0*.